### DISCUSSION TITLE
How to create access_token for a user without user interaction?

### DISCUSSION THREAD

**Discussion by liemle3893** [ory/hydra#2750] [Q&A] [answered]
Let's say I'm providing some sort of service and we need a way to create user access_token without any user interaction.
The standard way is

```
User -> Login Page (Create account if not exists) -> Consent page -> Access Token -> `/userinfo`
```

**What I want**:

```
User + (opaque token) -> API (this API will talk directly to Hydra) -> AccessToken -> `/userinfo`
```

Is there anyway to achive this?

**Comment by aeneasr** [comment, marked as answer]
Not right now. There is an open PR in ory/fosite for OAuth2 Token Exchange but is is currently blocked from merging due to legal issues.
